app/models/in_progress_form.rb

```diff
@@ -32,7 +34,7 @@ def metadata
     data = super || {}
     expires = updated_at || Time.current
```

Comment:
`expires` is a super confusing name for this variable, considering that word is used in like 15 other contexts in the same file. can we change it to something like `last_seen`?